content/en/blog/2025/2025-04-01-fediverse-security-fund.md

```diff
@@ -56,30 +56,31 @@ Projects should have either:
 
 The following projects are currently included and meet the above minimum requirements for vulnerability disclosure:
 
-* [Mastodon](https://github.com/mastodon/mastodon)
-* [NodeBB](https://nodebb.org/)
-* [Pleroma](https://git.pleroma.social/pleroma/pleroma)
-* [Pixelfed](https://github.com/pixelfed/pixelfed)
+* [Diaspora Federation](https://github.com/diaspora/diaspora_federation)
+* [Fedify](https://fedify.dev/)
 * [Forgejo](https://forgejo.org/)
-* [PeerTube](https://github.com/Chocobozzz/PeerTube)
-* [Hollo](https://docs.hollo.social/)
 * [Funkwhale](https://dev.funkwhale.audio/funkwhale/funkwhale)
-* [Diaspora](https://github.com/diaspora/diaspora)
+* [GoToSocial](https://gotosocial.org/)
+* [Hollo](https://docs.hollo.social/)
 * [Lemmy](https://github.com/LemmyNet/lemmy)
-* [Misskey](https://github.com/misskey-dev/misskey)
-* [Fedify](https://fedify.dev/)
 * [Manyfold](https://github.com/manyfold3d/manyfold)
+* [Mastodon](https://github.com/mastodon/mastodon)
+* [Misskey](https://github.com/misskey-dev/misskey)
+* [NodeBB](https://nodebb.org/)
+* [PeerTube](https://github.com/Chocobozzz/PeerTube)
+* [Pixelfed](https://github.com/pixelfed/pixelfed)
+* [Pleroma](https://git.pleroma.social/pleroma/pleroma)
 
 Don’t see the project you were expecting? Raise a ticket at [nivenly/community](https://github.com/nivenly/community/issues) and let’s have a discussion about adding it – we want to include open source Fediverse projects of all sizes!
 
 ### Projects Provisionally Included
 
 These projects are included within the Fund, however, their responsible security vulnerability disclosure practices need to be improved to continue to be considered eligible for the Fund beyond September 30, 2025.
 
+* [Discourse ActivityPub Plugin](https://github.com/discourse/discourse-activity-pub)
 * [Friendica](https://github.com/friendica/friendica)
 * [Hubzilla](https://framagit.org/hubzilla/core)
 * [Piefed](https://join.piefed.social/)
```

Comment:
Piefed can now be moved up too: https://codeberg.org/rimu/pyfedi/src/branch/main/SECURITY.md